Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -112,11 +114,12 @@ gdjs.LightRuntimeObjectPixiRenderer._verticesWithAngleComparator = function (
 gdjs.LightRuntimeObjectPixiRenderer._computeClosestIntersectionPoint = function (
   lightObject,
   angle,
-  polygons
+  polygons,
+  halfOfDiag
```

Comment:
`halfOfDiag` is not going to be sqrt(2)*radius anymore, because the boundary box could expand, and could easily become a rectangle at this point. So, this distance has to be passed now.